README.md

```diff
@@ -1,2 +1,75 @@
+
+> Note: If there is to be an extended Bootstrap transition committee to, for example, provide oversight in the organization's first year, those criteria must be defined as a new program in the proposed governance model. There is no extension of **this** specific group's life-cycle.
+
+## Definitions
+Throughout this repository we may use assorted acronyms and jargon to describe many complex concepts. This language is subject to change as we iterate on proposals. To help onboard and level-set, we will maintain an active list of commonly used language here. If you see domain specific language used in this repository that is not represented here, please add it!
+
+| Vocabulary | Definition |
+|---|---|
+| Bootstrap | This repo, team, and process. |
+| Ignition | The decision to move forward with the new governance |
+| Contributor | You, and anyone that interacts with this repo. |
+| TLCs | The Top Level Councils of the project – Currently the CPC and C3 (defined below) |
+| CPC | A new proposed Top Level Committee – The "Cross Project Council |
+| C3 | A new proposed Top Level Committee – The "Cross Community Council |
+| CNCF | The Cloud Native Computing Foundation (Prior Art) |
+| COC | Code of Conduct |
+| MVF | Minimum Viable Foundation. The bare minimum governance documentation to operate a function joint foundation |
+| RFC | Request For Comments. The feedback and iteration process followed in the `/programs` directory. |
+
+## Deliverables
+The deliverables from this body will be:
+
+  - [ ] A recommendations document to help inform the new Foundation's by-laws (Goal: November 1st)
+  - [ ] A recommendations document to help inform the TLCs' charters (Goal: ???)
+  - [ ] A collection of fully drafted governance documents required for Ignition. "Minimum Viable Foundation" (MVF). (Goal: March 1st)
+  - [ ] A collection of program, initiative, and governance recommendation documents not required for MVF. After Ignition these will be passed to the appropriate bodies (CPC, C3, etc.) to execute.
+
+## Operations
+
+Bootstrap follows a [Consensus Seeking][] decision making model.
+
+> Outstanding Questions:
+> If full consensus can not be achieved for a particular PR, what is the fallback voting structure?
+> If full consensus can not be achieved for Ignition, what is the fallback? 2/3rds supermajority of CommComm, TSC and TAC? Something else?
+
+The Code of Conduct explains the *bare minimum* behavior
+expectations the Node.js Foundation requires from all contributors.
+[Please read it before participating.][COC]
+
+To begin contributing to working groups and initiatives, start participating!
+
+The [`/programs`](./programs) directory contains instructions and templates for a standard RFC process for new Foundation programs proposals. To propose a new program that the joint Foundation may want to administer, please follow the directions in the `/programs` folder's [README](./programs/README.md).
```

Comment:
I think we should drop this part from this PR and move it over to https://github.com/nodejs/bootstrap/pull/23

better separation of concerns